### ISSUE TITLE
Is there a way to indent text in a cell

### ISSUE THREAD

**Issue by dgappc** [dhatim/fastexcel#309]
Excel has formatting to indent a cell by a number character (see image below). Is there a way to set this format using fastexcel?

Thanks

![image](https://github.com/dhatim/fastexcel/assets/124009353/21aa9c6e-41ab-4186-96c5-06f2ee4ce0d2)

**Comment by dgappc** [comment]
If indentation is difficult to support, a workaround can be to allow leading spaces when setting String values in a cell. 

Currently if I try to set a string with any leading spaces, the spaces get automatically removed by FastExcel. 

POI which we used previously does not behave that way

Thanks

**Comment by JakKowalCzyk** [comment]
@dgappc have you discovered how to set indentation with the fastexcel?

**Comment by dgappc** [comment]
No, I have not. You can use hard spaces "\u00a0" instead of regular space, but we don't like that, it interferes with search.

**Comment by JakKowalCzyk** [comment]
I see, thanks anyway
